k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -756,9 +752,10 @@ Upload any local messages that are marked as PENDING_UPLOAD (Drafts, Sent, Trash
         String storeUri = account.getStoreUri();
         if (ImapStore.isStoreUriImap(storeUri)) {
             ImapSyncInteractor syncInteractor = new ImapSyncInteractor(account, folderName, listener, this);
-            syncInteractor.performSync(flagSyncHelper, messageDownloader, syncHelper);
+            syncInteractor.performSync(flagSyncHelper, messageDownloader, notificationController, syncHelper);
```

Comment:
I believe I passed the classes that I had created by refactoring as method parameters. I'll change this.